rskj-core/src/test/java/co/rsk/peg/performance/RegisterFastBridgeBtcTransactionTest.java

```diff
@@ -0,0 +1,264 @@
+package co.rsk.peg.performance;
+
+import co.rsk.bitcoinj.core.*;
+import co.rsk.bitcoinj.script.FastBridgeRedeemScriptParser;
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptBuilder;
+import co.rsk.bitcoinj.store.BlockStoreException;
+import co.rsk.bitcoinj.store.BtcBlockStore;
+import co.rsk.core.RskAddress;
+import co.rsk.crypto.Keccak256;
+import co.rsk.peg.*;
+import org.ethereum.config.Constants;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
+import org.ethereum.core.Repository;
+import org.ethereum.crypto.ECKey;
+import org.ethereum.crypto.HashUtil;
+import org.ethereum.vm.exception.VMException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Ignore
+public class RegisterFastBridgeBtcTransactionTest extends BridgePerformanceTestCase {
+    private BtcTransaction btcTx;
+    private int blockWithTxHeight;
+    private boolean shouldTransferToContract =  true;
+    private BtcBlock blockWithTx;
+    private PartialMerkleTree pmt;
+    private Coin totalAmount;
+
+
+    @BeforeClass
+    public static void setupA() {
+        constants = Constants.regtest();
+        activationConfig = ActivationConfigsForTest.all();
+    }
+
+    @Test
+    public void registerFastBridgeBtcTransaction() throws VMException {
+        ExecutionStats stats = new ExecutionStats("registerFastBridgeBtcTransaction");
+        registerFastBridgeBtcTransaction_success(5000, stats);
+        registerFastBridgeBtcTransaction_surpasses_locking_cap(1000, stats);
+        BridgePerformanceTest.addStats(stats);
+    }
+
+    private void registerFastBridgeBtcTransaction_success(int times, ExecutionStats stats) throws VMException {
+
+        BtcECKey btcECKeyUserRefundAddress = new BtcECKey();
```

Comment:
Try to declare these fields as class attributes and set them only once, to avoid duplicating it on each test.